### ISSUE TITLE
Question: Synchronization Encryption

### ISSUE THREAD

**Issue by VGrol** [hluk/CopyQ#870]
I read earlier that the Synchronization plugin automatically encrypts the files.
I have not found a way to make this work. 
All synchronized tabs are stored as plain .txt files, I'd like to automatically encrypt them, what would be the best way to go about this?

**Comment by hluk** [comment]
> I read earlier that the Synchronization plugin automatically encrypts the files.

That's incorrect. There is separate [plugin for encryption](http://copyq.readthedocs.io/en/latest/password-protection.html) but it only works with separate items or whole tabs -- unfortunately, it currently works only with tabs that are not synchronized.

This actually is more of a bug than limitation since it's not even obvious that those plugins don't work together.

**Comment by cjohnsonuk** [comment]
Just to confirm if anyone is looking into this.  As of version v3.7.2 (2019-02-04)

1) if I encrypt a tab it asks for a password before showing me the content
2) if I synchronise/synchronize a tab to a folder it creates files in that folder ( a file and an associated .dat file)
3) if I encrypt and synchronise/synchronize a tab: 
  a) no new clips added to the tab get saved to the folder
  b) no new files added to the folder appear in the tab
  c) existing files are not encrypted

Your mileage may vary ...

If this could be made to work it would be great.  It would allow my team to share clips of code between PCs without resorting to email and filling up inboxes and with the added security of the encryption.
